lib/src/cupertino/cupertino_toolbar.dart

```diff
@@ -441,13 +470,70 @@ class _RenderIosPagedMenu extends RenderBox
       // Computes next button position.
       final nextButton = children.last;
       final nextButtonButtonParentData = nextButton.parentData as _IosPagerParentData;
-      nextButtonButtonParentData.offset = Offset(accumulatedWith, (height - nextButton.size.height) / 2);
+      nextButtonButtonParentData.offset = Offset(accumulatedWith, (effectiveHeight - nextButton.size.height) / 2);
 
       // Update current width.
       accumulatedWith += nextButton.size.width;
     }
 
-    size = Size(accumulatedWith, height);
+    size = constraints.constrain(Size(accumulatedWith, effectiveHeight));
+  }
+
+  @override
+  Size computeDryLayout(BoxConstraints constraints) {
```

Comment:
This looks very similar to what's above. Should any of this be de-duped?

Reply:
I think we might end up with less readable code, with lots of `if (!dry) {} ` checks...

Reply:
As we are now calling the regular layout in the popover, we don't need the dry layout anymore.